src/main/java/racingcar/model/domain/Victory.java

```diff
@@ -0,0 +1,35 @@
+package racingcar.model.domain;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Victory {
+
+    private static final String MAX_DRIVE_LENGTH = "MAX 값을 구할 수 없습니다. 비교할 수 있는 정수값인지 확인하세요";
+    private List<Car> victories;
+
+    private Victory(final List<Car> cars) {
+        victories = cars;
+    }
+
+    public static Victory maxCheck(final Cars cars) {
+
+        int max = maxDriveLength(cars);
+        return new Victory(cars.cars().stream()
+            .filter(car -> car.isMax(max, car))
+            .collect(Collectors.toList()));
```

Comment:
하나의 라인을 통해 모든 작업을 처리하려하는 경우 오히려 가독성이 떨어질 수 있을 것 같아요! 아래와 같이 라인을 나누어 작업을해보면 어떨까요?

```suggestion
        final List<Car> victories = cars.cars().stream()
            .filter(car -> car.isMax(max, car))
            .collect(Collectors.toList());
        return new Victory(victories);
```

Reply:
`cars.cars()` 를 줄여주면서 자연스럽게 알려주신 모습으로 바뀌었어요. 감사합니다. :)!!